recipes/Data-Prep-Kit/intro/dpk_intro_1_ray.ipynb

```diff
@@ -0,0 +1,1410 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "841e533d-ebb3-406d-9da7-b19e2c5f5866",
+      "metadata": {
+        "id": "841e533d-ebb3-406d-9da7-b19e2c5f5866"
+      },
+      "source": [
+        "# Data Prep Kit Demo 1 - Ray Version\n",
+        "\n",
+        "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/sujee/granite-code-cookbook/blob/recipe/data-prep-kit-intro/recipes/Data-Prep-Kit/intro/dpk_intro_1_ray.ipynb)\n",
```

Comment:
Again, please find and fix ALL references to your personally GitHub account. There are 40 of them across the notebooks, according to `grep`.

All the `https://raw.githubusercontent.com/sujee/granite-code-cookbook/recipe/` should be `https://raw.githubusercontent.com/ibm-granite-community/granite-code-cookbook/refs/heads/main/recipes`. Note the `recipe` -> `recipes`. I understand that when this change is made, the notebooks' `wget` commands won't work in Colab until this PR is merged to `main`. 

A way to fix that problem is use two PRs, one which stages assets and the second which adds the notebooks.

If you do this, discuss with the team where to stage such assets. Using a `media` subdirectory might be fine, but the team might prefer to put all this stuff in one place, for example an `assets` folder at the top-level.